monitoring/uss_qualifier/scd/data_interfaces.py

```diff
@@ -1,12 +1,30 @@
-from typing import Optional
+from typing import Optional, List, Dict
 from monitoring.monitorlib.typing import ImplicitDict
 from monitoring.monitorlib.scd_automated_testing.scd_injection_api import InjectFlightRequest
-import datetime
+from monitoring.monitorlib import fetch
+from monitoring.uss_qualifier.rid import Severity
 
-class SCDTestIssue(ImplicitDict):
-    ''' A class to hold a message that the test executor can provide to the USS in cases when the USS provides a response that is not same as the expected result. '''
-    timestamp: Optional[str]
+class Result(object):
```

Comment:
There are many ways that enums could be treated/expressed, but let's be consistent.  We previously discussed [this](https://github.com/interuss/dss/pull/657#discussion_r775062762) and [this](https://github.com/interuss/dss/pull/657#discussion_r773356609).

Reply:
Thanks! I have now made it consistent with the other PR data definitions.